src/react/PopupMenu.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PopupMenuItem from './PopupMenuItem';
+
+class PopupMenu extends React.Component {
+	render() {
+		const {children = [], onMenuItemClick, position = {}, relative} = this.props;
+		const style = relative ? {left: position.offsetX, top: position.offsetY} : {};
```

Comment:
In Angular, it is `position.x` and `position.y`. Better keep on consistency.

Reply:
I agree, we will change this